security/keycloak-webapp/src/main/docker/Dockerfile.jvm

```diff
@@ -0,0 +1,34 @@
+####
```

Comment:
Both docker files are just copy and paste from http module I guess.

They are quite legacy ones, current Docker files are based on `registry.access.redhat.com/ubi8/ubi-minimal:8.1`.
Check https://quarkus.io/guides/getting-started#bootstrapping-the-project generated project to see the latest version of docker files.

Reply:
Updated both files.

Reply:
We should probably remove all the `src/main/docker` directories. I originally left them there, in hope that we'll eventually use them, but that doesn't seem to be the case.

Reply:
Depends what "Docker based build for OpenShift" will translate into.
So I would keep them in place for few days/weeks before we know for sure the is no use for them.